tests/app/forms/field_handlers/test_string_handler.py

```diff
@@ -21,3 +22,36 @@ class TestForm(Form):
     assert isinstance(form.test_field, StringField)
     assert form.test_field.label.text == textfield_json["label"]
     assert form.test_field.description == textfield_json["guidance"]
+
+
+def test_get_length_validator():
+    string_handler = StringHandler({}, {}, AnswerStore(), {})
+
+    validator = string_handler.get_length_validator
+
+    assert isinstance(validator, validators.Length)
+
+
+def test_get_length_validator_with_message_override():
+    answer = {
+        "validation": {"messages": {"MAX_LENGTH_EXCEEDED": "The message is too long!"}}
+    }
+    string_handler = StringHandler(
+        answer,
+        {"MAX_LENGTH_EXCEEDED": "This is the default max length message"},
+        AnswerStore(),
+        {},
+    )
+
+    validator = string_handler.get_length_validator
+
+    assert validator.message == "The message is too long!"
+
+
+def test_get_length_validator_with_max_length_override():
+    answer = {"max_length": 30}
+
+    string_handler = StringHandler(answer, {}, AnswerStore(), {})
```

Comment:
I think the StringHandler class only requires `answer`, so we should be able to drop the empty dicts and AnswerStore from all the places it's instantiated.